.NET/Microsoft.Recognizers.Text.DateTime/Constants.cs

```diff
@@ -14,25 +14,22 @@ public static class Constants
         public const string SYS_DATETIME_SET = "set";
         public const string SYS_DATETIME_DATETIMEALT = "datetimealt";
 
-        //key
-        public const string TimexKey = "timex";
-        public const string CommentKey = "Comment";
-        public const string ModKey = "Mod";
-        public const string TypeKey = "type";
-        public const string IsLunarKey = "isLunar";

```

Comment:
Some of these should stay here. Only the ones that show up in model results should move to your new class.

Reply:
For the ones that stay here, maybe we need to rename them and follow the code standard of this file.

Reply:
fixed